app/views/apps/index.html.slim

```diff
@@ -10,6 +10,17 @@
       i.icon.far.fa-plus-square
       = t('apps.new.title')
 
+#search.row.mb-5
```

Comment:
It looks so werid on search section layout, why place a empty `.col-sm-3` and the search button is too large.

and missing the i18n :D